doc/whats_new/v0.21.rst

```diff
@@ -137,6 +139,15 @@ Support for Python 3.4 and below has been officially dropped.
   affects all ensemble methods using decision trees.
   :pr:`12344` by :user:`Adrin Jalali <adrinjalali>`.
 
+:mod:`sklearn.linear_model`
+...........................
+
+- |Feature| :class:`linear_model.LogisticRegression` and
+  :class:`linear_model.LogisticRegressionCV` now support Elastic-Net penalty,
+  with the 'saga' solver. :issue:`11646` by :user:`Nicolas Hug <NicolasHug>`.
+
+- |Fix| Fixed a bug in the 'saga' solver where the weights would not be
+  correctly updated in some cases :issue:`11646` by `Tom Dupre la Tour`_.
```

Comment:
Thanks for the review @TomDLT , please feel free to modify this entry for a more accurate description. I don't understand the bug fix in details.

Reply:
Add a . after "some cases"